### PR TITLE
Update Overview of APIs for April 2024

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:
* **Overview of top-level feature areas** - new heading above the table.  Lists **Environment options** row instead of **User data** row.
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3126#overview-of-top-level-feature-areas
   * [After/Live](https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#overview-of-top-level-feature-areas)
* **Environment options** - new top-level section.
   * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3126#environment-options
   * [After/Live](https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#environment-options)
   * **User data** - demoted from a top-level section to a subsection.
      * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3126#user-data
      * [Before/Live](https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#user-data)
   * **Runtime selection** - new subsection.
      * https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3126#runtime-selection
      * [After/Live](https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#runtime-selection)

This PR updates Overview of APIs for April 2024 promotions to Stable in Release SDK:
https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#10247835

Outline of headings: 
```
# Overview of WebView2 features and APIs

## Overview of top-level feature areas - NEW SECTION HEADING

## Main classes: Environment, Controller, and Core

## Web/native interop
#### Host/web object sharing
#### Script execution
#### Web messaging
#### Script dialogs
#### Shared buffer

## Browser features
#### Printing
#### Cookies
#### Image capture
#### Downloads
#### Permissions
#### Context menus
#### Status bar
#### User Agent
#### Autofill
#### Audio
#### Hit-testing of mouse-clicks in regions
#### Swipe gesture navigation
#### Enable or disable the browser responding to accelerator keys (shortcut keys)
#### Fullscreen
#### PDF toolbar
#### Theming
#### Language
#### New window
#### Close window
#### Document title
#### Favicon
#### Security and privacy
###### Tracking prevention
###### SmartScreen
###### Custom crash reporting
#### Browser extensions

## Process management
#### Frame process info

## Navigate to pages and manage loaded content
#### Manage content loaded into WebView2
#### Navigation history
#### Navigation kind
#### Block unwanted navigating
#### Navigation events
#### Manage network requests in WebView2
#### Custom scheme registration
#### Client certificates
#### Server certificates
#### Launch an external URI scheme

## iframes

## Authentication

## Rendering WebView2 in non-framework apps
#### Sizing, positioning, and visibility
#### Zooming
#### Rasterization scale
#### Focus and tabbing
#### Parent window
#### Keyboard accelerators
#### Default background color

## Rendering WebView2 using Composition
#### Connecting to the visual tree
#### Forwarding input
#### Drag and drop
#### Accessibility

## Environment options - NEW SECTION HEADING
#### User data - DEMOTED FROM TOP-LEVEL HEADING
###### Multiple profiles
###### Delete a profile
#### Runtime selection - INCOMING NEW SECTION

## Performance and debugging
#### Memory usage target

## Chrome DevTools Protocol (CDP)

## See also
```
AB#50082263

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -2749,12 +2743,77 @@ Analyze and debug performance, handle performance-related events, and manage mem
...
 * `ICoreWebView2_6` interface:
    * [ICoreWebView2_6::OpenTaskManagerWindow method](/microsoft-edge/webview2/reference/win32/icorewebview2_6#opentaskmanagerwindow)
 
 ---
 
 
+<!-- ------------------------------ -->
+#### Select browser preview channels for testing
+
+The Runtime selection feature supports prerelease testing and flighting scenarios.  You can specify `ReleaseChannels` to choose which channels are searched for during environment creation, and `ChannelSearchKind` to select a search order.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:2756]:
```suggestion
The Runtime selection feature supports prerelease testing and selfhosting scenarios.  You can specify `ChannelSearchKind` to select a search order and `ReleaseChannels` to choose which channels are searched for during environment creation.
```

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -2724,6 +2717,7 @@ See also:
 Analyze and debug performance, handle performance-related events, and manage memory usage to increase the responsiveness of your app.
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:2717]: I don't think runtime selection should be in performance and debugging. 
My suggestion is to create:
- a new Environmental Options section, and we put both User Data and Runtime selection under it.
- Create a new Runtime Selection section

**Review by victorhuangwq — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]:
```
<!-- ====================================================================== -->
## Environment options

**Subsections below:**
* [User data](#user-data)
* [Runtime selection](#runtime-selection)

...

<!-- ------------------------------ -->
#### Runtime selection

The Runtime Selection feature supports prerelease testing and self-hosting.
To specify a search order for browser preview channels, use the
`CoreWebView2EnvironmentOptions.ChannelSearchKind` property.  
To specify which browser preview channels are searched for during environment creation, use the
`CoreWebView2EnvironmentOptions.ReleaseChannels` property.

See also:
* [Test upcoming APIs and features](../how-to/set-preview-channel.md)
<!-- todo: link, after PR https://github.com/MicrosoftDocs/edge-developer/pull/3053 is merged:
* [Prerelease testing using preview channels](../how-to/prerelease-testing.md)
* [Self-host by deploying preview channels](../how-to/self-hosting.md)
-->
```

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md]:
```
The Runtime selection feature supports prerelease testing and self-hosting.
When creating a WebView2 environment:
* To specify a search order for browser preview channels, use the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property.
* To specify which browser preview channels are searched for, use the `CoreWebView2EnvironmentOptions.ReleaseChannels` property.

```

**Review by peiche-jessica — APPROVED** [review]

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**